tools/pipelines/templates/include-install-pnpm.yml

```diff
@@ -36,6 +36,8 @@ steps:
 
 - task: Bash@3
   displayName: Install and configure pnpm
+  env:
+    COREPACK_DEFAULT_TO_LATEST: 0
```

Comment:
@tylerbutler I think this is the correct place, but since it wasn't here before it means a potential change in behavior? But maybe our pinning of pnpm makes it redundant?

Reply:
Hmmm, if it wasn't applied before probably best to remove it. The packageManager field should be enough.

Reply:
Argh, didn't push before merging this. I'll do a separate PR to take it out.

Reply:
https://github.com/microsoft/FluidFramework/pull/15610